web/src/test/java/it/pagopa/selfcare/dashboard/web/model/mapper/InstitutionMapperTest.java

```diff
@@ -216,7 +216,7 @@ void toResource_AuthOnDifferentInstitutionPendingStatus() {
     @Test
     void toResourceNull() {
         // given and when
-        InstitutionResource institutionResource = InstitutionMapper.toResource(null);
+        InstitutionResource institutionResource = InstitutionMapper.toResource((InstitutionInfo) null);
```

Comment:
using the mapstruct mapper we can avoid doing unit tests on InstitutionMapperTest